src/package.json

```diff
@@ -11,5 +11,6 @@
 	},
 	"main": "./bundles/umd/base.js",
 	"browser": "./index.js",
+	"module": "./index.js",
```

Comment:
Module is used by some build tools instead of main to correctly infer where the ESM code lives.